shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -294,7 +294,7 @@ bool OffScreenRenderWidgetHostView::HasFocus() {
 }
 
 bool OffScreenRenderWidgetHostView::IsSurfaceAvailableForCopy() {
-  return GetDelegatedFrameHost()->CanCopyFromCompositingSurface();
+  return get_delegated_frame_host()->CanCopyFromCompositingSurface();
```

Comment:
last one 😸

Reply:
Thanks for catching all of these!

Updated in 9031bd4d7c78209259148b83a1f2be16ca2f715e